Hashes for 4.7.3

natetharp opened the pull request:
Added hashes for WP version 4.7.3

philipjohn commented:
Thanks @natetharp !

In checking this, it looks like the hash for `wp-settings.php` is missing. Also, there's a closing PHP tag in the file, which we should remove.

Did you use the latest version of `hashes-generator.php` to generate these hashes?

natetharp commented:
Np @philipjohn. I did use the `hashes-generator.php` from the current plugin for these, and there was an unlabeled hash at the beginning. I removed that hash and it appeared to work fine on my WP installation, which is now surprising.

I've added that hash back for `wp-settings.php` and removed the closing PHP tag that the `hashes-generator.php` spit out. It again appeared to perform as expected in my WP installation.

mattyrob commented:
Patch looks good to me - add hash file for 4.7.3
